practice-project-api/.env

```diff
@@ -0,0 +1 @@
+PORT=8000
```

Comment:
You have to `git rm .env` to ensure that this is removed from the repo since you've committed one already